modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-scaling-worker-nodes_{context}"]
+= Scaling worker nodes
+
+[role="_abstract"]
+Worker nodes can be scaled manually if you do not want to configure node autoscaling.
+
+. Procedure
+
+. To find the ID of the machine pool you want to scale and verify the current number of replicas, enter the following command to get a list of the machine pools in a cluster. Each cluster has a default machine pool that is created when you create a cluster.
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID        AUTOSCALING   REPLICAS    INSTANCE TYPE  LABELS    TAINTS   AVAILABILITY ZONES
+default   No            2           m5.xlarge                        us-east-1a
+mp1       No            2           m5.xlarge                        us-east-1a
+----
+
+. To change the number of replicas, or scale, of the machine pool, enter the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> --replicas=<number_worker_nodes> <machinepool_ID>
+----
++
+.Example
++
+[source,terminal]
+----
+rosa edit machinepool --cluster=mycluster --replicas=3 default
+----
+
+. To see that the change has taken effect, enter the following command:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name>
+----
++
+The response output shows the number of worker nodes, or replicas, as `Compute` nodes.
+
+. (Optional) To view this change in the link:https://cloud.redhat.com/openshift[OCM console]:
```

Comment:
s/. (Optional) To view/. Optional: To view

Reply:
Done.